### PR TITLE
Cleaned up sdk/math.ooc

### PR DESCRIPTION
**Pull request by marcusnaslund**
Again, with `?w=1` this is a very small PR.
Only style fixes to make magic happy, and some comments removed.

### CODE REVIEW

File: source/sdk/math.ooc [hunk 1]
```diff
@@ -86,36 +86,31 @@ extend Int {
...
-	minimum: static func ~two(first: This, second: This) -> This {
+	minimum: static func ~two (first: This, second: This) -> This {
 		first < second ? first : second
 	}
 	// TODO: Avoid using this, consider removing it.
-	minimum: static func ~multiple(value: This, values: ...) -> This {
+	minimum: static func ~multiple (value: This, values: ...) -> This {
 		values each(|v|
-			if ((v as This*)@ < value)
+			if ((v as This *)@ < value)
```
Comment (davidhesselbom) [inline source/sdk/math.ooc]: If you've done this before, I've missed it. Does a space really belong here?

Comment (marcusnaslund) [inline source/sdk/math.ooc]: It looks weird, I know, but magic complains if it's not there.
I had to do the same thing for the `net` classes.

Comment (davidhesselbom) [inline source/sdk/math.ooc]: @thomasfanell Can we make magic understand this isn't a multiplication operator?

Comment (davidhesselbom) [inline source/sdk/math.ooc]: Actually, I found plenty of examples of `type*` in another project, and magic didn't mind that. None of them were `This*`, though. Magic bug?

Comment (marcusnaslund) [inline source/sdk/math.ooc]: You're right, magic wants a space after every `This` for some reason. It's not because magic believes it's a multiplication operator (since it doesn't complain about there being no space between `*` and `)`).

Comment (thomasfanell) [inline source/sdk/math.ooc]: Yeah I can fix that.

Comment (thomasfanell) [inline source/sdk/math.ooc]: And it's not a bug, it's just related to the fact that ```*``` is disregarded since it can mean different things, as we see here. The violated rule here is the one that deals with keyword spacing (in this case, ```This```). I've added ```*``` to the ignore list for this particular rule and should have a release up in a few minutes.

Comment (davidhesselbom) [inline source/sdk/math.ooc]: Great, thanks.

Comment (marcusnaslund) [inline source/sdk/math.ooc]: I will fix this then.

**Comment by marcusnaslund** [comment]
There we go.

File: source/sdk/math.ooc [hunk 2]
```diff
@@ -86,34 +86,29 @@ extend Int {
...
 		)
 		value
 	}
-	minimum: static func ~two(first: This, second: This) -> This {
+	minimum: static func ~two (first: This, second: This) -> This {
 		first < second ? first : second
 	}
 	// TODO: Avoid using this, consider removing it.
-	minimum: static func ~multiple(value: This, values: ...) -> This {
+	minimum: static func ~multiple (value: This, values: ...) -> This {
```
Comment (davidhesselbom) [inline source/sdk/math.ooc]: Doesn't belong in this PR, but, when you have the time, could you investigate the possibility of removing this and the corresponding `maximum` function (the ones with `...`)?

Comment (marcusnaslund) [inline source/sdk/math.ooc]: It's on my list :)

Comment (marcusnaslund) [inline source/sdk/math.ooc]: Are we not interested in `VarArgs` at all?

Comment (davidhesselbom) [inline source/sdk/math.ooc]: I don't think they can be used without leaking memory. They also have absolutely 0 type safety - the arguments in there could be *anything*.

Comment (marcusnaslund) [inline source/sdk/math.ooc]: So let's throw them out of the SDK, then?

Comment (davidhesselbom) [inline source/sdk/math.ooc]: I guess so. But not in this PR :)

Comment (marcusnaslund) [inline source/sdk/math.ooc]: Once they're not used anymore, I mean.
